### PR TITLE
Removes some logging code from the app delegate.

### PR DESCRIPTION
**Pull request by diegoreymendez**
Removes some logging code from the app delegate.

**Test 1:**
- Place a breakpoint in `configureLogging`.
- Launch the app and make sure the method is called.

**Test 2:**
- Place a breakpoint in `fileLogger`.
- Launch the app and make sure the file logger is created.

**Test 3:**
- Place a breakpoint in the last line of the app delegate's `application:didFinishLaunchingWithOptions:`.
- Launch the app.
- When the breakpoint is hit, type `po [self.logger getLogFilesContentWithMaxSize:5000]`
- Make sure the returned string makes sense by comparing it to the log in the console.

### CODE REVIEW

File: WordPress/Classes/Utility/Logging/WPLogger.h [hunk 1]
```diff
@@ -0,0 +1,28 @@
+//
+//  WPLogger.h
+//  WordPress
+//
+//  Created by Diego E. Rey Mendez on 3/26/15.
```
Comment (SergioEstevao) [inline WordPress/Classes/Utility/Logging/WPLogger.h]: This headers need to go out :)

File: WordPress/Classes/Utility/Logging/WPLogger.m [hunk 2]
```diff
@@ -0,0 +1,115 @@
+//
```
Comment (SergioEstevao) [inline WordPress/Classes/Utility/Logging/WPLogger.m]: Headers need to be removed

**Comment by SergioEstevao** [comment]
Good work here @diegoreymendez I'm really enjoying seeing the AppDelegate get trimmed.

:shipit:

File: WordPress/Classes/Utility/Logging/WPLogger.h (hunk 1 above)
Comment (diegoreymendez) [inline WordPress/Classes/Utility/Logging/WPLogger.h]: Oh my... always forget.

**Comment by diegoreymendez** [comment]
Thanks!

**Pull request merged**